api/billingd/service/service.go

```diff
@@ -952,8 +953,11 @@ func (s *Service) handleUsage(ctx context.Context, cus *Customer, product Produc
 			product.Key + "_usage":      total,
 			product.Key + "_units":      product.Units,
 			product.Key + "_free_quota": product.FreeQuotaSize,
+			"grace_period_start":        s.analytics.FormatUnix(cus.GracePeriodStart),
+			"invoice_period_end":        s.analytics.FormatUnix(cus.InvoicePeriod.UnixEnd),
+			"invoice_period_start":      s.analytics.FormatUnix(cus.InvoicePeriod.UnixStart),
+			"subscription_status":       cus.SubscriptionStatus,
```

Comment:
Maybe also snag the _account status_? `cus.AccountStatus()`

Reply:
Another one, maybe snag number of dependents? You'd have to do,
```
deps, err := s.cdb.CountDocuments(ctx, bson.M{"parent_key": cus.Key})
if err != nil {
	return nil, err
}
```

Reply:
okay, nice. made me realize a consolidation that could happen here for fewer calls. pushed

Reply:
Mentioning one more in case it makes sense: `pb.Usage` has a daily cost param (`Cost`), which is derived from other things: https://github.com/textileio/textile/blob/master/api/billingd/service/service.go#L621

Reply:
slipped it in https://github.com/textileio/textile/pull/445/commits/7f60f75b8903cad54a2eb457d8c527e831624ce7

Reply:
Sweet!